ethcore/light/Cargo.toml

```diff
@@ -12,16 +12,16 @@ ethcore = { path = ".."}
 parity-bytes = "0.1"
 ethcore-transaction = { path = "../transaction" }
 ethereum-types = "0.4"
-memorydb = "0.2.1"
-patricia-trie = "0.2"
+memorydb  = "0.3.0-beta"
```

Comment:
why such strange versioning?

versions below 1.0 better to name just numerically, they are all betas

Reply:
To be clear: once https://github.com/paritytech/parity-common/pull/67 is reviewed but before it lands, I'll fix the versions and publish the proper `0.3.0` versions.